### ISSUE TITLE
[CI] test failing related due to Breakpoint

### ISSUE THREAD

**Issue by MarcusDenker** [guillep/pharo-core#16245]
**Bug description**
The PR https://github.com/pharo-project/pharo/pull/16177 was merged with tests failing.

- ClyInstallMetaLinkPresenterTest>>testInstallSelectedMetalink
- ClyInstallMetaLinkPresenterTest>>testInstallSelectedMetalinkActionButton
- DebugPointTest>>testTranscriptDebugPoint

And two code quality checks:

- ProperMethodCategorizationTest>>testNoLeadingOrTrailingSpacesInCategoryNames
- DebugPointTest>>#testBreakDebugPointOnVariableDetectsNewVariableAccessAfterCompilingNewMethod

**Comment by adri09070** [comment]
Where can I see the details of the failing tests ?

**Comment by MarcusDenker** [comment]
E.g here:

https://ci.inria.fr/pharo-ci-jenkins2/blue/organizations/jenkins/Test%20pending%20pull%20request%20and%20branch%20Pipeline/detail/PR-16177/10/tests

<blockquote><div><strong><a href="https://ci.inria.fr/pharo-ci-jenkins2/blue/organizations/jenkins/Test%20pending%20pull%20request%20and%20branch%20Pipeline/detail/PR-16177/10/tests">Jenkins</a></strong></div></blockquote>

**Comment by adri09070** [comment]
From what i understand, this is because I set an object-centric debug point on variable writes in this test: `DebugPointTest>>#testObjectCentricBreakOnceOnVariable`

As it migrates the target object to an anonymous subclass, it instruments super calls with another metalink. However, when I uninstall my metalink, from what I see in the chain call, it should uninstall the metalinks that instrument the super calls but it actually doesn't.

I've debugged it the whole day and I might have found a solution. I'll try it tomorrow
